source/cli.txt

```diff
@@ -39,6 +39,22 @@ Installation
 
 .. include:: /includes/install-realm-cli-v2.rst
 
+Authentication
```

Comment:
Hmm, I think I like this on the CLI page - not so much as a separate page within the CLI docs. Can we kill the `cli/auth-with-api-token` page and just redirect to here? (I wonder if there's a reason we had it separate before, like maybe the TSEs link to it - or maybe it's just a leftover from before we had generated CLI docs/better CLI docs.)